src/legacy/ui/ui_render/bootstrap/template.js.hbs

```diff
@@ -46,7 +46,7 @@ if (window.__kbnStrictCsp__ && window.__kbnCspNotEnforced__) {
     function createJavascriptElement(path) {
       var dom = document.createElement('script');
 
-      dom.setAttribute('async', '');
+      dom.setAttribute('defer', true);
```

Comment:
Just a minor thing I realised.
I think in the majority of the cases it should not make a difference, but according the documentation the `defer` attribute is a boolean attribute https://html.spec.whatwg.org/multipage/scripting.html#attr-script-defer.

The boolean attributes should not have any `true` or `false`. The `false` is represented by the absence of the tag, and the `true` is represented by the presence of the tag with an empty string or with the attribute string itself https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute. It's also mentioned here (https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute).

So both `dom.setAttribute('defer', '');` or `dom.setAttribute('defer', 'defer');` are a better option to assure everything works as expected 😄